editor/components/post-permalink/style.scss

```diff
@@ -34,3 +38,20 @@
 		@include long-content-fade( $size: 20% );
 	}
 }
+
+.editor-post-permalink__edit-form {
+	width: 100%;
+}
+
+.editor-post-permalink__prefix,
+.editor-post-permalink__suffix {
+	color: $dark-gray-300;
+}
+
+.editor-post-permalink__edit {
+	margin: 0 5px;
+}
+
+.editor-post-permalink__save {
+	float: right;
```

Comment:
Maybe some flex instead of `float`, to avoid the inevitable future headaches:

![image](https://user-images.githubusercontent.com/1779930/37850464-928c0f80-2eb1-11e8-9f19-3291d79e1ec3.png)

Reply:
💪🏻